mipengine/node/tasks/views.py

```diff
@@ -66,9 +66,10 @@ def create_pathology_view(
     )
     columns.insert(0, DATA_TABLE_PRIMARY_KEY)
 
+    # TODO Now the data_models require a version to access the proper table with data.
```

Comment:
nit: Ticket for this?

Reply:
https://team-1617704806227.atlassian.net/browse/MIP-506